openqabot/openqa.py

```diff
@@ -56,3 +58,16 @@ def get_jobs(self, data: Data):
             logger.exception(e)
             raise e
         return ret
+
+    @lru_cache(maxsize=256)
+    def is_devel_group(self, groupid: int) -> bool:
+        ret = None
+
+        try:
+            ret = self.openqa.openqa_request("GET", f"job_groups/{groupid}")
+        # TODO: correct handling
+        except Exception as e:
+            logger.exception(e)
+            raise e
+
+        return ret[0]["parent_id"] == 9  # magic number of parent "Development" group
```

Comment:
We can do better :)

Reply:
How ? `ret[0]["name"] == ""Development"` ... same thing

Reply:
Well, using "Development" as part of the name is certainly a convention. But we shouldn't hardcode a "magic number" in the source code. I think reading out the parent job group names is feasible but if you would prefer to prevent an API call every time then I suggest to move this ID to a configuration variable that has "9" as default